pkg/indexer/indexer.go

```diff
@@ -33,8 +33,7 @@ func RegisterAll(ctx context.Context, mgr manager.Manager, logger *zap.Logger) e
 		NewAtlasFederatedAuthBySecretsIndexer(logger),
 		NewAtlasDatabaseUserBySecretsIndexer(logger),
 		NewAtlasDatabaseUserByCredentialIndexer(logger),
-		NewAtlasDatabaseUserByProjectsRefIndexer(logger),
-		NewAtlasDatabaseUserByExternalProjectsRefIndexer(logger),
```

Comment:
I do not understand why we remove this.

The fact that the deployment can be managed independently does not change the fact that db users can also be managed independently and will need their indexes.

Reply:
Wait I was confused, these are not the ones affecting the Database User reconciler. Those are on lines 34 and 35, so it is all fine.

Sorry for the noise.

Reply:
those do not affect the db users, I see that now